quantmsio/commands/convert/idxml.py

```
@@ -56,3 +60,312 @@ def convert_idxml_file(
+            mzml_file_paths,
+            mzml_map,
+            logger,
+        )
+
+        try:
+            parquet_file = _convert_single_idxml(
+                idxml_file, mzml_file, temp_dir, i, logger
+            )
+            parquet_files.append(parquet_file)
+        except Exception as e:
+            logger.error(f"Failed to convert {idxml_file}: {e}")
```

Ruff (0.14.0)

Use `logging.exception` instead of `logging.error`

Replace with `exception`

(TRY400)